app/src/main/java/de/westnordost/streetcomplete/quests/segregated/AddSegregated.java

```diff
@@ -0,0 +1,67 @@
+package de.westnordost.streetcomplete.quests.segregated;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.text.TextUtils;
+
+import java.util.List;
+import java.util.Map;
+
+import javax.inject.Inject;
+
+import de.westnordost.streetcomplete.R;
+import de.westnordost.streetcomplete.data.meta.OsmTaggings;
+import de.westnordost.streetcomplete.data.osm.SimpleOverpassQuestType;
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder;
+import de.westnordost.streetcomplete.data.osm.download.OverpassMapDataDao;
+import de.westnordost.streetcomplete.quests.AbstractQuestAnswerFragment;
+import de.westnordost.streetcomplete.quests.powerpoles_material.AddPowerPolesMaterialForm;
+
+public class AddSegregated extends SimpleOverpassQuestType {
+
+	@Inject
+	public AddSegregated(OverpassMapDataDao overpassServer) {
+		super(overpassServer);
+	}
+
+	@Override
+	protected String getTagFilters() {
+		return
+			"ways with " +
+				"(" +
+				"(highway = path and bicycle = designated and foot = designated)" +
+				" or (highway = footway and bicycle = designated)" +
+				" or (highway = cycleway and foot = designated)" +
```

Comment:
Why only designated?

Reply:
because (at least in my experience) highway=footway + bicycle=yes is always segregated=no

Is it possible that bicycle has dedicated part of path and the way is not dedicated to usage by cyclists?

Reply:
I can change that but I am curious how correctly tagged "highway=footway + bicycle=yes + segregated=yes" or "highway=cycleway + foot=yes + segregated=yes" looks like.

Reply:
Wait… we indeed [discussed that already](https://github.com/westnordost/StreetComplete/issues/527#issuecomment-324309986).
The sign, @ENT8R linked or included there – which link is apparently gone – is this one:
![grafik](https://user-images.githubusercontent.com/11966684/42734436-1b606220-8844-11e8-996f-8ffb86eb4d80.png)

Just had to re-read what I've written there and seems I got to this, in the end:
> TL;DR: Quest should also ask for foot/bicycle=yes values as a legal designation via street sign […] is not needed for segregation. And pictures should be shown instead of street signs for the same reason.

So, keep it simple: Is it segregated or not?

Reply:
AFAIK the issue was that `designated` implies a legal meaning, and, as such, `yes` can be used if there is no explicit sign.

**The images you look for were posted here**: https://github.com/westnordost/StreetComplete/issues/527#issuecomment-325822251

Reply:
>  The sign, @ENT8R linked or included there – which link is apparently gone – is this one:

Yes, I know about such places. But every single one that I encountered with this sign of equivalent is not segregated. That is why I asked for a photo of such location.

Reply:
To add my two cents: 
At least in Germany, and apparently documented that way in the German wiki (didn't check), there exists the opinion that only cycleways which are compulsory to use should be tagged with cycleway=designated. And I think this has been documented like this already a long time because I remember when I started with OSM, I decided to rather not tag any cycleway as `designated` (from satellite imagery) but rather as just `yes` because I was not sure if it was compulsory to use. See also this forum topic https://forum.openstreetmap.org/viewtopic.php?id=62974 to see that this opinion is still around.
Long story short, and wrong as this "only in Germany, `designated` has a special additional meaning" may be, it has been practised like this for a long time, so chances are that a lot of these situations are around.

If the solution here should be to include all `yes` as well then, ... well, this is something we should discuss.

Reply:
I guess it also never does any harm to include `yes`. These are only a few extra roads then, ….

Reply:
"only a few extra roads" - my city has 970+ highway=footway + bicycle=yes without segregated, all of them are segreagted=no ( http://overpass-turbo.eu/s/Ajo )

Reply:
Well, then perhaps not include footways with `cycleway=yes` after all. The Germans just need to get their act together and fix their tagging practice.